src/org/omegat/core/dictionaries/LingvoDSL.java

```diff
@@ -170,6 +170,8 @@ private static String replaceTag(final String line) {
         TAG_REPLACEMENTS.put(Pattern.compile("\\[\\[(?<content>.+?)]]"), "&#91;${content}&#93;");
         TAG_REPLACEMENTS.put(Pattern.compile(Pattern.quote("\\[")), "&#91;");
         TAG_REPLACEMENTS.put(Pattern.compile(Pattern.quote("\\]")), "&#93;");
+        // lang tag
+        TAG_REPLACEMENTS.put(Pattern.compile("\\[lang id=(.+?)]"), "");
```

Comment:
Wouldn't there be a closing `[/lang]` tag as well?

Also [these docs](http://lingvo.helpmax.net/en/troubleshooting/dsl-compiler/dsl-tags/) say that the attribute is `name` not `id`. We can just catch anything by doing `\\[lang[^]]*]`.

Reply:
Also we handle `lang` in `ignoreTags` below. If we're going to handle it here instead, we should remove it from `ignoreTags` for clarity.

Reply:
Does this handle closing tag as well?
`TAG_REPLACEMENTS.put(Pattern.compile("\\[/?lang[^]]*]"), "");`

Reply:
Also adding test and passed it.